metricbeat/module/kubernetes/apiserver/_meta/fields.yml

```diff
@@ -4,10 +4,6 @@
     Kubernetes API server metrics
   release: ga
   fields:
-    - name: request.client
```

Comment:
Not a blocker: There is no way to retrieve this information in the latest versions? This would be a nice visualisation to show the distribution of requests per client type.

Reply:
No unfortunately client was removed in version 1.18